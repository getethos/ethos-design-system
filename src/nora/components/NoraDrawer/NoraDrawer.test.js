import React from 'react'
import { NoraDrawer } from './index'
import { render } from '@testing-library/react'

describe('NoraDrawer', () => {
  it('default rendering', () => {
    const onDismissFn = jest.fn()
    const tree = render(
      <NoraDrawer
        isOpen={true}
        position="right"
        labelCopy="Order Evidences"
        closeCopy="Cancel"
        onDismiss={onDismissFn}
      >
        {' '}
        <p>This is a test</p>
        <button>A button</button>
      </NoraDrawer>
    )
    expect(tree).toMatchSnapshot()
  })
})
