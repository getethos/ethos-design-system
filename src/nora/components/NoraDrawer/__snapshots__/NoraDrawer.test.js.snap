// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoraDrawer default rendering 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div
      aria-hidden="true"
    />
    <div
      id="drawer-root"
    >
      <div
        aria-hidden="false"
        class="DrawerWrapperActive"
        data-testid="base-drawer-container"
      >
        <div
          aria-hidden="false"
          aria-label="drawer"
          class="Container Open Right"
          role="dialog"
        >
          <header
            class="Header"
          >
            <button
              class="Close"
            >
              Cancel
            </button>
            Order Evidences
          </header>
           
          <p>
            This is a test
          </p>
          <button>
            A button
          </button>
        </div>
      </div>
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
